src/components/annotator/CategoriesList/SaveMenu/ExportAnnotationsAsLabeledInstancesMenuItem.tsx

```diff
@@ -28,7 +28,7 @@ export const ExportAnnotationsAsLabeledInstancesMenuItem = ({
     popupState.close();
     handleCloseMenu();
 
-    if (!annotations) return;
+    if (!annotations.toLocaleString) return;
```

Comment:
I think your autocomplete misled you here 😂

Reply:
yes it did!😂